src/utilities/cities/index.ts

```diff
@@ -0,0 +1,24 @@
+import { CITIES_DATA } from '../../common/cities';
+
+import { State, getStates } from '../states';
+
+const sortAlphabetically = (cityA: string, cityB: string) => cityA.localeCompare(cityB);
+
+export function getCities({ state: stateFilter }: { state?: Partial<State> } = {}): string[] {
```

Comment:
I really don't know if it is good, but you can type the returned cities based on the received state. When you fetch cities data, you can generate the correct types. What do you think about it?